internal/app/service/service_handler_events.go

```diff
@@ -109,3 +111,47 @@ type RemoveIssueRepositoryFromServiceEvent struct {
 func (e *RemoveIssueRepositoryFromServiceEvent) Name() event.EventName {
 	return RemoveIssueRepositoryFromServiceEventName
 }
+
+// OnServiceCreate is a handler for the CreateServiceEvent
+// Is creating a single default priority for the default issue repository
+func OnServiceCreate(db database.Database, e event.Event) {
+	//@todo configure somewhere
+	var defaultPrio int64 = 100
```

Comment:
Same here: Do we want to fix that todo now?

Reply:
Implemented.